mono/metadata/class-internals.h

```diff
@@ -1469,6 +1469,347 @@ mono_class_init_checked (MonoClass *klass, MonoError *error);
 MonoType*
 mono_class_enum_basetype_internal (MonoClass *klass);
 
+// Eventually, changes within MONO_JIT_ICALLS might require revising MONO_AOT_FILE_VERSION.
+// i.e. if AOT persists enum/int instead of string. For now, this is not the case.
```

Comment:
These should go to the previous jit-icall-reg.h file, and that file should be included into this file.

Reply:
Ok. I didn't like the name of it, but a separate file I do like.
I'll do jit-icall-reg.h as you say and it can renamed if that was just quick typing.
The include will probably at the end "like this".